.imgbotconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "schedule": "weekly",
+  "ignoredFiles": [
+    "public/images/overlay/*"
```

Comment:
I think it has to be /*/**

Reply:
![image](https://github.com/dotabod/frontend/assets/71287790/353a0f6a-b723-4fa7-ac0c-0b66d3c0d0dd)
Ya either /*/** or /** - definitely wrong rn tho.